mailbox.go

```diff
@@ -164,7 +165,7 @@ func newMailbox(s *Server, name, nsName string, size int) (*Mailbox, error) {
 				cancel()
 				return err != nil
 			})
-		if err != nil {
+		if err != nil && err != registry.ErrNotOwner {
```

Comment:
```suggestion
		// Ingnore ErrNotOwner because most likely the previous owner panic'ed or exited badly. 
		// So we'll ignore the error and let the mailbox creator retry later.  We don't want to panic
		// in that case because it will take down more mailboxes and make it worse. 
		if err != nil && err != registry.ErrNotOwner {
```